scripts/pd.json

```diff
@@ -6508,6 +6916,7 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
+          "description": "The count of the corresponding schedule commands which PD sends to each TiKV instance",
```

Comment:
This description is wrong.

Reply:
It has been changed to "The heartbeat latency of each TiKV instance in .99"